packages/arb-bridge-eth/src/utils.ts

```diff
@@ -34,8 +34,8 @@ export const getL2ChainId = (): Bytes => {
     }
   }
 
-  if (network == "rinkeby") return Bytes.fromByteArray(Bytes.fromHexString("0x066EEB"));
   if (network == "goerli") return Bytes.fromByteArray(Bytes.fromHexString("0x066eed"));
+  if (network == "sepolia") return Bytes.fromByteArray(Bytes.fromHexString("0x66eee"));
```

Comment:
`0x66eee` -> `0x066eee` should resolve the error that arb-bridge-eth-sepolia subgraph is throwing

Reply:
it is syncing!! thank you